README.md

```diff
@@ -112,7 +112,9 @@ class BadInstagramCloneApp extends Component {
   }
 
   takePicture() {
-    this.camera.capture()
+    const options = {};
+    //options.location = ...
+    this.camera.capture({metadata: options})
```

Comment:
what is the expected shape of metadata? what is the reason not to simply have metadata be a boolean?